cypress/support/commands.js

```diff
@@ -1,194 +1,204 @@
-					'trakt-api-version': Cypress.env('trakt_api_version'),
-					'trakt-api-key': Cypress.env('client_id'),
-				},
-				failOnStatusCode: false,
-			}).then($response => {
-				expect($response.status).to.be.gt(200);
-			});
-		});
-	});
-});
-
-Cypress.Commands.add('deleteAnyActiveCheckins',()=>{
-	cy.request({
-		method: 'DELETE',
-		url: `https://api.trakt.tv/checkin`,
-		headers: {
-			'Content-Type': 'application/json',
-			Authorization: `Bearer ${Cypress.env('access_token')}`,
-			'trakt-api-version': Cypress.env('trakt_api_version'),
-			'trakt-api-key': Cypress.env('client_id'),
-		},
-		failOnStatusCode: false,
-	}).then($response => {
-		expect($response.status).to.be.eq(204);
-	});
-
-});
-
-Cypress.Commands.add('deleteComment',id=>{
-	cy.request({
-		method: 'DELETE',
-		url: `https://api.trakt.tv/comments/${id}`,
-		headers: {
-			'Content-Type': 'application/json',
-			Authorization: `Bearer ${Cypress.env('access_token')}`,
-			'trakt-api-version': Cypress.env('trakt_api_version'),
-		  'trakt-api-key': Cypress.env('client_id'),
-		},
-		failOnStatusCode: false,
-	  }).then($response =>{
-		expect($response.status).to.be.eq(204)
-	  });
-	});
-
-
-
-
+/// <reference types="cypress" />
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('verifyCode', (code, verificationUrl) => {
+	cy.visit(`https://trakt.tv/auth/signin`)
+	cy.get('#user_login').type(Cypress.env('user').username)
+	cy.get('#user_password').type(Cypress.env('user').password)
+	cy.contains('Sign in').click()
+
+	cy.visit(verificationUrl)
+	cy.get('#code').type(code)
+	cy.contains('Continue').click()
+	cy.get('#auth-form-wrapper').should('be.visible')
+	cy.get('input[name="commit"]').contains('Yes').click()
+	cy.contains('Woohoo!').should('be.visible')
+})
+
+Cypress.Commands.add('getAccessToken', () => {
+	cy.request({
+		method: 'POST',
+		url: 'https://api.trakt.tv/oauth/device/code',
+		headers: {
+			'Content-Type': 'application/json',
+		},
+		failOnStatusCode: false,
+		body: {
+			client_id: Cypress.env('client_id'),
+		},
+	}).then($response => {
+		expect($response.status).to.be.eq(200)

```

Comment:
Remove assertions from commands. We do the assertions in the tests. Here we just do the actions and return the response on which we either assert or just confirm here that the operation went well with status OK and that's it.

In the tests we test this same scenario but with assertions. The point of the custom commands is to make them reusable although we have to test this precise same scenario in the tests. In this manner, we avoid the test dependencies between each other and passing of parameters.